proto/regen/ecocredit/v1alpha1/events.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+
+package regen.ecocredit.v1alpha1;
+
+option go_package = "github.com/regen-network/regen-ledger/x/ecocredit";
+
+// EventCreateClass is an event emitted when a credit class is created.
+message EventCreateClass {
+
+    // class_id is the unique ID of credit class.
+    string class_id = 1;
+
+    // designer is the designer of the credit class.
+    string designer = 2;
+}
+
+// EventCreateBatch is an event emitted when a credit batch is created.
+message EventCreateBatch {
+
+    // class_id is the unique ID of credit class.
+    string class_id = 1;
```

Comment:
What is the reason that these identifiers (class_id) are string and not integer?

I guess for `batch_denom` its useful to have as a string, as that's the same type as denoms in the broader cosmos sdk, but for `class_id` how do you see these ID's being generated? In my head, this would have been an integer identifier, indexed (as in a relational database) and auto-incrementing

Reply:
An integer converted to a string and so that it's obviously part of denom. We can reassess later